python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -370,17 +360,19 @@
 # %%
 #
 # The Sobol' indices confirm the previous analyses, in terms of ranking of the most influent variables. We also see that five variables have a quasi null total Sobol' indices, that indicates almost no influence on the wing weight.
-# There is no discrepancy between first order and total Sobol' indices, that indicates no or very low interaction between the variables in the variance of the output.
+# There is no discrepancy between first order and total Sobol' indices, that indicates no or very low interaction between the variables in the variance of the output. 
+# As the most important variables act only through decoupled first degree contributions, the hypothesis of a linear dependence between the input variables and the weight is legitimate, which explains why both squared SRC and Taylor give the exact same results even if the first one is based on a :math:`\mathcal{L}^2` linear approximation and the second one is based on a linear expansion around the mean value of the input variables.

```

Comment:
these lines are way too long and make the review difficult

Reply:
> these lines are way too long and make the review difficult

The line has been splitted.